.github/actions/common/ActionBase.ts

```diff
@@ -54,10 +54,13 @@ export class ActionBase {
 
 		// All indicated labels must be present
 		if (this.labels) {
+			if (this.labels?.length > 2  && this.labels?.startsWith('"') && this.labels?.endsWith('"')) {
```

Comment:
What if; instead of supporting quoting of the entire string, the individual labels could be quoted?  And, if there are already quotes, not adding additional quoting to the query (on line 63 here)?

Reply:
individual quotes doesn't work with yaml syntax. Those were the first things I tried.